.github/workflows/ci-plugin-server.yml

```diff
@@ -91,8 +91,9 @@ jobs:
                   sudo bash -c 'echo "127.0.0.1 kafka zookeeper" >> /etc/hosts'
                   ping -c 1 kafka
                   ping -c 1 zookeeper
+
             - name: Start Kafka, ClickHouse, Zookeeper, Object Storage
-              run: docker-compose -f docker-compose.dev.yml up -d zookeeper kafka clickhouse object_storage &
```

Comment:
Here we are still selectively starting some services as there's some no-sense going on in this file and we spin up Redis and PostgreSQL separately